index.js

```diff
@@ -224,11 +228,12 @@ class Ky {
 	constructor(input, options = {}) {
 		this._retryCount = 0;
 		this._input = input;
+		const headers = isObject(this._input.headers) ? this._input.headers : {};
```

Comment:
This can be simplified ! just checking what's the practice here.

Possible 
```
  this._options = {
    headers: mergeHeaders(this._input.headers || {}, options.headers);
  }
```